src/components/ui/ToggleButton.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import { withStyles } from '@material-ui/core/styles';
+import { green } from '@material-ui/core/colors';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import Switch from '@material-ui/core/Switch';
+
+const GreenSwitch = withStyles({
+  switchBase: {
+    color: green[300],
+    '&$checked': {
+      color: green[500],
+    },
+    '&$checked + $track': {
+      backgroundColor: green[500],
+    },
+  },
+  checked: {},
+  track: {},
+})(Switch);
+
+function ToggleButton() {
+  const [checked, setChecked] = useState(false);
+
+  const toggleChecked = () => {
+    setChecked((state) => !state);
+  };
+  return (
+    <FormControlLabel
+      control={<GreenSwitch checked={checked} onChange={toggleChecked} />}
+      label="Dark mode"
```

Comment:
Instead to hardcode labels, you can create props in order to setting up this out of the component for its reutilization.

If won't to reuse this component, could be a good idea to rename to a more descritible component such as `ToogleDarkMode`